pkg/workload/rand/rand.go

```diff
@@ -56,6 +64,17 @@ func init() {
 	workload.Register(randMeta)
 }
 
+// defaultRandomSeed sets the `defaultSeed` package variable if it's
+// not yet set. This makes it possible for every run of the `rand`
+// workload to use a different seed by default
+func defaultRandomSeed() int64 {
```

Comment:
Clarification - why is `sync.Once` required over simply binding `randutil.NewPseudoSeed()` to `&g.seed`on L92